00 - Notes/เว็บ 2.0 คืออะไร.md

```diff
@@ -0,0 +1,7 @@
+# Web 2.0 คืออะไร ?
+**Web 2.0** เกิดขึ้นเมื่อต้นปี 2000 และยังเป็นรูปแบบเว็บไซต์ที่มีจำนวนมากที่สุดในปัจจุบัน เป็นเว็บไซต์ที่ผู้ใช้งาน นอกเหนือจากการอ่านข้อมูลภายในเว็บไซต์ได้แล้วยังสามารถสร้าง Content ต่างๆ ได้อีกด้วย โดยรูปแบบของ Web 2.0 จะเป็นแบบ Read-Write & Interaction เช่น โพสต์ข้อความ, กดไลค์,  เขียนบล็อก, แชร์รูปภาพ, แชร์วีดีโอ, คอมเม้นท์,  ส่งข้อความ ฯลฯ จะไม่ใช่แค่การเสพข้อมูลเพียงอย่างเดียวเหมือนอย่างสมัย **[[เว็บ 1.0 คืออะไร|Web 1.0]]** เรียกการสื่อสารแบบนี้ว่าสื่อสารแบบสองทาง หรือ Two Way Communication  จุดเด่นของ Web 2.0 คือ การขับเคลื่อนด้วยข้อมูลบนเว็บไซต์ที่เหล่าผู้ใช้งานสร้างขึ้นมา โดยรูปแบบ Model การติดต่อสื่อสารจะหว่าง Client และ Server จะเป็นแบบ Client and Server Model
```

Comment:
Web 2.0 ไม่ใช่ Website

Reply:
Interactive?

Reply:
"โดยรูปแบบ Model การติดต่อสื่อสารจะหว่าง Client และ Server จะเป็นแบบ Client and Server Model" ทั่วไปอ่านไม่เข้าใจ

Reply:
ผ่านตัวกลาง ?